tests/src/JIT/Regression/JitBlue/GitHub_18259/GitHub_18259.cs

```diff
@@ -0,0 +1,619 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+// Test case for issues with a missing check for ZeroOffsetFldSeq values on LclVar reads
```

Comment:
A little more detail here and/or on the specific cases would be informative for future reference.

Reply:
Without the fix several cases fail because value numvereing gives us the old value instead of the modified value.
The original bug was submitted by the Fuzzlyn C# generator program.

I wrote up this test case to cover alot of the different cases.
Without my fix this test case prints:

D:\Public\Bugs\GitHub_18259>VN_RefArrayStruct.exe
Failed - Test_e0_S2_F3_F1() - 640
Failed - Test_e0_S2_F4_F1() - 960
Failed - Test_e1_S2_F3_F1() - 640
Failed - Test_e1_S2_F4_F1() - 960
Failed

Reply:
I also have a smaller test case that I can add as well.